DocumentsFromSnapshotMigration/src/test/java/org/opensearch/migrations/bulkload/ParallelDocumentMigrationsTest.java

```diff
@@ -80,22 +77,24 @@ public void testDocumentMigration(
             .withAllTracking();
 
         try (
-            var esSourceContainer = new PreloadedSearchClusterContainer(
```

Comment:
Just a question - are we still using this container anywhere after your change?

Reply:
We aren't using the preloaded container anymore.